project_wbs/view/project_project_view.xml

```diff
@@ -214,7 +214,6 @@
                 </xpath>
                 <xpath expr='//field[@name="state"]' position="after">
                     <field name="stage_id" widget="statusbar" clickable="True"/>
-                    <field name="stage_id" class="oe_inline" required="1"/>
```

Comment:
This field was added as a workaround to be able to create projects from form view. The clickable statusbar doesn't work with the on_change.